app/src/controllers/log/storageUtils.js

```diff
@@ -17,36 +17,23 @@
 import { getStorageItem, updateStorageItem } from 'common/utils';
 import { MARKDOWN } from 'common/constants/logViewModes';
 import {
-  DEFAULT_WITH_ATTACHMENTS,
```

Comment:
Remove it from constants too. It isn't used somewhere else.

Reply:
Thank you. Removed